db/file_indexer.h

```diff
@@ -58,7 +58,8 @@ class FileIndexer {
                    std::vector<FileMetaData*>* const files);
 
   enum {
-    kLevelMaxIndex = std::numeric_limits<int32_t>::max()
+    // MSVC version 1800 still does not have constexpr for ::max()
+    kLevelMaxIndex = INT32_MAX
```

Comment:
@siying Let's chat quickly after you done with the review. I sent you personal email.